wunderio/next-drupal-starterkit
# Add .gitattributes file; update .gitignore

## Changes proposed in this PR:

Adds `.gitattributes` file, which tells git explicitly how to handle line ending normalisation for various file types, and by doing that, improves cross-platform developer experience.

## How to test:

1. Have a Windows developer make changes to some file.
2. See that there is no git warning about line endings when committing.
3. Also see that the line endings in the repository are indeed normalized to LF-only (Unix format `\n` instead of the Windows format `\r\n`)
